src/engine/enginebuffer.cpp

```diff
@@ -595,29 +595,31 @@ void EngineBuffer::slotPassthroughChanged(double enabled) {
 }
 
 // WARNING: This method runs in both the GUI thread and the Engine Thread
-void EngineBuffer::slotControlSeek(double change) {
-    doSeek(change, SEEK_STANDARD);
+void EngineBuffer::slotControlSeek(double fractionapPos) {
+    doSeekFractional(fractionapPos, SEEK_STANDARD);
 }
 
 // WARNING: This method runs from SyncWorker and Engine Worker
-void EngineBuffer::slotControlSeekAbs(double abs) {
-    doSeek(abs / m_file_length_old, SEEK_STANDARD);
+void EngineBuffer::slotControlSeekAbs(double playPosition) {
+    doSeek(playPosition, SEEK_STANDARD);
```

Comment:
this changes doSeek from seeking to a proportional position to an absolute position.  Please do not change the API because this may break something elsewhere.